mlflow/projects/databricks.py

```diff
@@ -299,12 +302,22 @@ def __init__(self, databricks_run_id, mlflow_run_id, databricks_job_runner):
         self._mlflow_run_id = mlflow_run_id
         self._job_runner = databricks_job_runner
 
-    def _print_description(self):
+    def _print_description_and_log_tags(self):
         eprint("=== Launched MLflow run as Databricks job run with ID %s. Getting run status "
                "page URL... ===" % self._databricks_run_id)
         run_info = self._job_runner.jobs_runs_get(self._databricks_run_id)
         jobs_page_url = run_info["run_page_url"]
         eprint("=== Check the run's status at %s ===" % jobs_page_url)
+        set_tag(MLFLOW_DATABRICKS_RUN_URL, jobs_page_url)
```

Comment:
As per offline discussion - we should call `tracking.get_service().log_tag(self._mlflow_run_id, key, value)` here to avoid creating a new run.